ethcore/engines/authority-round/src/lib.rs

```diff
@@ -2871,7 +2888,7 @@ mod tests {
 				"validators": {
 					"list" : ["0x1000000000000000000000000000000000000001"]
 				},
-	            "blockRewardContractTransition": "7",
+			"blockRewardContractTransition": "7",
```

Comment:
Please revert the faulty identation

Reply:
Wasn't it faulty before? I replaced the spaces with tabs.
Or do you mean it shouldn't be in this PR because it's unrelated to the change? I'm happy to revert it, of course, if you prefer.

Reply:
It was correct before AFAIU, otherwise please convince me :)

Reply:
I thought we were using only tabs? These two lines were the only result when I searched for four spaces, so I assume they should have been tabs instead. Also, that's how line 2886 above is indented, which should be at the same level as this one.

Reply:
Yeah, if those are spaces that is wrong (missed that) but the indentation level should match `validators` on the line above.

Sorry for being a PITA.

Reply:
Ah, you're right, of course, sorry!
I updated it. Please double-check that I got it right this time. :sweat_smile:

Reply:
Looks good :+1: